CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8.8)
+cmake_minimum_required (VERSION 3.0)
```

Comment:
> For Debian it should be actually okay because it only effects oldoldstable.

There is a Debian version that is even more ancient than `oldstable` 😱… Anyway, even the [official Debian release page](https://wiki.debian.org/DebianReleases) does not list `oldoldstable` under current releases. For now I just removed the commit that includes the version bump. I guess the current CMake files should still work, even if we use CMake 2.8 to generate the build system.